src/pages/Registry/index.js

```diff
@@ -146,24 +146,71 @@ export default function Registry() {
         tableHeadings: tableHeadings,
         tableValues: rewardConfigTableValues
       });
+
+      const hookConfigTableValues = [
+        {
+          key: 'Governor Address',
+          value: <Tooltip title="Governor address"><span>{governorAddress}</span></Tooltip>,
```

Comment:
Can we give some additional detail in the tooltips?

Reply:
I have now added "Hook" to the description. Can we add some more?